### PR TITLE
De-emphasize timestamp in flame chart hovercard

### PR DESCRIPTION
**Pull request by bduffany**
image

### CODE REVIEW

File: app/flame_chart/flame_chart.tsx [hunk 1]
```diff
@@ -548,14 +548,14 @@ class HoveredBlockInfo extends React.Component<{ buildDuration: number }, Hovere
...
-            <span className="data">{(ts + dur) / MICROSECONDS_PER_SECOND}</span> seconds
-          </div>
-          <div>
+
+          <div className="duration">
             <span className="data">{dur / MICROSECONDS_PER_SECOND}</span> seconds total (
             <span className="data">{percentage}</span> of total build duration)
           </div>
+          <div>
+            @ {ts / MICROSECONDS_PER_SECOND} s &ndash; {(ts + dur) / MICROSECONDS_PER_SECOND} s
```
Comment (siggisim) [inline app/flame_chart/flame_chart.tsx:557]: can we use your nifty on decimal rounding func on these (including duration) and truncate them all at 3 decimals?

if not, totally fine

**Review by siggisim — APPROVED** [review]
lgtm!

File: app/flame_chart/flame_chart.tsx (hunk 1 above)
Comment (bduffany) [reply app/flame_chart/flame_chart.tsx]: Good idea, that'll help reduce information overload. Done!

**Comment by siggisim** [comment]
lgtm!

**Pull request merged**